decidim-admin/app/forms/decidim/admin/scope_form.rb

```diff
@@ -27,7 +27,7 @@ def scope_type
       private
 
       def code_uniqueness
-        return unless organization && organization.scopes.where(code: code).where.not(id: id).any?
+        return unless organization&.scopes&.where(code: code)&.where&.not(id: id)&.any?
```

Comment:
All this operators except for the first one are unnecessary, right?

Reply:
If `organization` is `nil`, then `organization&.scopes` is `nil`, and `organization&.scopes.where(...)` will raise a `NoMethodError`. I guess if we use the `&.` operator we're doomed to use it in all the line 😕

Reply:
You're right! I liked the older style better. :/

Reply:
I guess we can tweak the config and allow both styles? I'll try.

Reply:
If we want to allow both styles I guess we can disable the rule.

But I'm fine with these `&.` chains if it's currently the only way to enforce consistency.

Reply:
Yeah @mrcasals, just ignore me. This is just me being defensive about a code style that I've never seen. I'll just get used to it :)

Reply:
It would be nice if rubocop forced us to change `participatory_process && participatory_process.scope` to `participatory_process&.scope`, but when we need to go deeper it allowed us to leave it as `organization && organization.scopes.where(code: code).where.not(id: id).any?` for example (in order to avoid multiple `&.` calls) 😕 

I checked the docs but could not find any option to allow this though 😭

Reply:
Yeah, that's the style I would prefer too. We can make a feature request :)

Reply:
Hm, checking the tests for the cop, it looks like all these `&.` were manually added:

https://github.com/bbatsov/rubocop/blob/1317b8919c86cb8b207b73f09943c3a677c35f8c/spec/rubocop/cop/style/safe_navigation_spec.rb#L962-L994

What should we do?

Reply:
Seems like a bug in the cop... We could leave as is, or make these cases a little more verbose, like:

```ruby
return unless organization

organization.scopes.where(code: code).where.not(id: id).any?
```

Would that be equivalent and make `rubocop` happy?